src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/ClassWriter.java

```diff
@@ -1365,23 +1365,23 @@ void write(ClassWriter writer) {
             }
         }
 
-        static class SameLocals1StackItemFrame extends StackMapTableEntry {
+        static class SameLocals1StackItemFrame extends StackMapTableFrame {
```

Comment:
static final class SameLocals1StackItemFrame extends StackMapTableFrame {

Reply:
We will refrain from touching this for now for parity with mainline.